### PR TITLE
replace humanized_opening_hours with JS library

### PR DESCRIPTION
**Pull request by remi-dupre**
Replace [humanize_opening_hours](https://github.com/rezemika/humanized_opening_hours) with a wrapper over [the Javascript library](https://github.com/opening-hours/opening_hours.js) using PyMiniRacer. The Python library only supported a partial syntax for opening_hours fields and came with some bugs.

### CODE REVIEW

File: idunn/blocks/opening_hour.py [hunk 1]
```diff
@@ -82,38 +75,27 @@ def parse_time_block(cls, es_poi, lang, raw):
...
-    # This is why we replace the timezone info until this problem is fixed in the library.
-    try:
-        nt = oh.next_change(dt=poi_dt.replace(tzinfo=None))
-    except HOHError:
-        logger.info(
-            "HOHError: Failed to compute next transition for poi %s",
-            es_poi.get("id"),
-            exc_info=True,
-        )
+    next_transition = oh.next_change()
```
Comment (amatissart) [inline idunn/blocks/opening_hour.py:88]: `is_open` and `next_change` should be evaluated on the exact same datetime `poi_dt`, to avoid edge cases when this function would be called at the exact moment when the state is changing, and the output would then be consistent.

Improbable, but definitely possible.

File: tests/test_opening_hours.py [hunk 2]
```diff
@@ -400,21 +401,145 @@ def test_oh_with_only_closed_rules():
 
 
 @freeze_time("2019-07-01T08:00:00")
-def test_oh_24_7_variant():
-    oh_block = get_moscow_poi("Apr 1-Sep 30: off")
-    assert oh_block is None
```
[the remark this answers is not in the document]
Comment (remi-dupre) [reply tests/test_opening_hours.py:405]: I think this test was an exact duplicate of [previous test](https://github.com/QwantResearch/idunn/pull/154/files#diff-01531746634b3dd0c0b81940cb4d80c7L398-L399), I'm not sure what was the intention, maybe something like `Jan 1-Dec 31: on` then ?

[the remark this answers is not in the document]
Comment (amatissart) [reply tests/test_opening_hours.py:405]: Arf, ok, I hadn't see the previous test. Nevermind, let's ignore this.

File: Pipfile [hunk 3]
```diff
@@ -25,7 +25,6 @@ python-redis-rate-limit = "*"
 python-json-logger = "*"
 lark-parser = "<0.7"
 astral = "==1.*"
```
Comment (amatissart) [inline Pipfile:27]: `lark-parser` and `astral` are subdependencies of `osm_humanized_opening_hours` that had to be pinned here. 
They should be removed now.
